types/index.d.ts

```diff
@@ -198,6 +198,8 @@ declare module "jspdf" {
     image?: HTMLOptionImage;
```

Comment:
The `enableLinks` and `pagebreak` options have no effect. We should remove them, as well. Also remove the HTMLOptionsPageBreak interface.